pyquil/api/_quantum_computer.py

```diff
@@ -101,28 +101,17 @@ def get_isa(self, oneq_type='Xhalves', twoq_type='CZ'):
         return self.device.get_isa(oneq_type=oneq_type, twoq_type=twoq_type)
 
     @_record_call
-    def run(self, executable: BinaryExecutableResponse, classical_addresses=None) -> np.ndarray:
+    def run(self, executable: BinaryExecutableResponse) -> np.ndarray:
         """
         Run a quil executable.
 
         :param executable: The program to run. You are responsible for compiling this first.
-        :param classical_addresses: The indices within regions of classical memory to report.
-            These don't necessarily correspond to qubit indices; rather they are the second
-            argument to any MEASURE instructions you've added to your program. If you don't
-            provide a specific collection, we will automatically report on all the locations
-            mentioned in your program.
-        :return: A numpy array of shape (trials, len(classical_addresses)) that contains 0s and 1s
+        :return: A numpy array of shape (trials, len(ro-register)) that contains 0s and 1s
```

Comment:
Better make this `-` a `_` unless you want to get accused of promoting Lisp.

Reply:
I sortof did this on purpose since it isn't a python thing ... its a quil thing